src/styles/global.css

```diff
@@ -1,92 +1,43 @@
 @import 'leaflet/dist/leaflet.css';
+@import "tailwindcss";
+@plugin "@tailwindcss/typography";
 
-/* Add your global styles and theme variables here */
+@theme {
+  --color-josh-bg: hsl(210deg, 30%, 8%);
+  --color-josh-text: hsl(0deg, 0%, 100%);
+  --color-josh-primary: hsl(333deg, 100%, 52%);
+  --color-josh-secondary: hsl(180deg, 100%, 40%);
+  --color-josh-accent: hsl(250deg, 100%, 70%);
+  --color-josh-gray: hsl(210deg, 15%, 60%);
```

Comment:
I mean we don't need to literally copy some of these CSS values. This is kind of ridiculous